.github/actions/upload-test-artifacts/action.yml

```diff
@@ -129,3 +129,13 @@ runs:
         retention-days: 14
         if-no-files-found: error
         path: test/**/*.xml
+
+    - name: Store Usage Logs on Github
+      uses: actions/upload-artifact@v3
+      if: inputs.use-gha
+      with:
+        # Add the run attempt, see [Artifact run attempt]
+        name: usage-log-runattempt${{ github.run_attempt }}-${{ inputs.file-suffix }}.zip
+        retention-days: 14
+        if-no-files-found: error
```

Comment:
can this be changed to ignore or warn?

Reply:
Yeah, let's stick with a warning here https://github.com/actions/upload-artifact/blob/main/action.yml#L16. I kind of copy this line from above